.github/workflows/maven.yml

```diff
@@ -23,7 +24,14 @@ jobs:
       with:
         java-version: '17'
         distribution: 'temurin'
-        cache: 'maven'
+    - name: Cache local Maven repository
```

Comment:
I think it would be usefull to as the java-people to adjust their default to include target files as well.

Reply:
That would be great, but I suspect they will not consider Eclipse target files important enough for the whole java-world.
But it's worth to ask. What would also help if there would be an option to adjust the pattern of included files.

Reply:
> What would also help if there would be an option to adjust the pattern of included files.

maybe this is a more acceptable change for generic java world then...

Reply:
> > What would also help if there would be an option to adjust the pattern of included files.
> 
> maybe this is a more acceptable change for generic java world then...

I assume that this is at least more likely to happen.
Do you want to ask the setup-java action devs? Otherwise I can do that later.

Reply:
Created https://github.com/actions/setup-java/issues/345.